LevelComponents/Room.cpp

```diff
@@ -1015,7 +1015,8 @@ namespace LevelComponents
     /// </summary>
     void Room::SetLayerDataPtr(int LayerNum, int dataPtr)
     {
-        assert(!(LayerNum & 0xFFFFFFFC) /* LayerNum must be within range [0, 4) */);
```

Comment:
Let's convert this into a debug warning and remove the comments

Reply:
Done.